practitioner/src/main/java/org/smartregister/extension/rest/PractitionerDetailsResourceProvider.java

```diff
@@ -43,28 +61,22 @@
 
 public class PractitionerDetailsResourceProvider implements IResourceProvider {
 
-    @Autowired
-    private IFhirResourceDao<Practitioner> practitionerIFhirResourceDao;
+    @Autowired private IFhirResourceDao<Practitioner> practitionerIFhirResourceDao;
```

Comment:
Please review indentation here.
I believe access modifier should be with the object declaration i.e. The next line. Feel free to share your thoughts @dubdabasoduba . This applies to other objects declaration as well.